### ISSUE TITLE
Jonah copied my idea

### ISSUE THREAD

**Issue by TheLightMC** [Shawckz/AutoSQL#1]
Me: "I'm going to go make something for AutoSQL"
Jonah: "That's a horrible idea"
Me: *Shows screenshot*
Jonah: "Wow just stop"
Me: "K :("
*One day later*
Jonah: "I made an AutoSQL"
Jonah: *Show's me the same implementation as I had*
Me: I hate you.

**Comment by Shawckz** [comment]
Liar.
No proof.
Bullshit.

**Comment by TheLightMC** [comment]
PROOOF!!!!!! http://chatformat.com/1wVOBqvk/

**Comment by Fireflies** [comment]
>basically impossible due to how difficult it is to serialize shit 

Difficult my fucking ass. Just check if the object is a valid POJO and you're 50% of the way there. Simple annotations from the `javax` package could easily provide things like data type and max values.

**Comment by SpongyBacon** [comment]
didnt read but can tell its static aboose

**Comment by Fireflies** [comment]
PS: Extra shame on you. Bukkit even has an ORM included. Look at [Avaje Ebean](http://www.avaje.org/).

**Comment by Shawckz** [comment]
@Fireflies wat r u talking about
@SpongyBacon Where?

**Comment by Shawckz** [comment]
@Fireflies That was a while ago before i thought of different ways to do it
chilllllllllllll

**Comment by SpongyBacon** [comment]
shawcking comments signify static aboose

**Comment by Fireflies** [comment]
tldr: it's not remotely difficult and Bukkit/Avaje Ebean already does it.
